File: node-client/scripts/update_pod.js
const Api = require('kubernetes-client');

//Get credentials (token and certificate) and api endpoints
const api = new Api.Api(Api.config.getInCluster());

//UPDATE POD
const podJSONupdated = {
    "apiVersion": "v1",
    "kind": "Pod",
    "metadata": {
      "name": "mynginxpod"
    },
    "spec": {
      "containers": [
        {
          "name": "nginx",
          "image": "nginx:1.8",
          "ports": [
            {
              "containerPort": 80
            }
          ]
        }
      ]
    }
  };
  api.group('v1').ns('default').pods('mynginxpod').patch({ body: podJSONupdated });
  

File: python-client/scripts/update_pod.py
from kubernetes import client, config

config.load_incluster_config()
v1 = client.CoreV1Api()

pod=client.V1Pod()
pod.metadata=client.V1ObjectMeta(name="mypod")
container=client.V1Container(name="nginx")
container.image="nginx:1.8"
spec=client.V1PodSpec(containers=[container])
pod.spec = spec

v1.patch_namespaced_pod(namespace="default", name="mypod", body=pod)